### DISCUSSION TITLE
Es buena idea cambiar a Marlin en mi impresora KP3s?

### DISCUSSION THREAD

**Discussion by AlanBrito22** [community/community#132101] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Muy buenas noches.
Tengo la impresora Kingroom KP3s 3.0 desde hace meses, pero recientemente he visto videos de como hacen cambio de filamento y que es mejor este firmware para una impresora. He visto guías por parte de YouTube y la pagina oficial de kingroom, pero en ninguna detallan como hacer ese cambio de filamento e incluso, después de instalar el Marlin no dicen nada de que laminador usar. La verdad no quiero arriesgar mucho la impresora. 
Se me olvido decir que he visto comentarios de al momento de instalar Marlin, el extrusor deja de sacar filamento y tuvieron que recurrir al firmware por defecto. Honestamente solo quiero saber como hacer un cambio de filamento ya sea con el firmware original o no, la impresora la uso solo para proyectos escolares pero me gustaría empezar a utilizarla para hacer negocio.
Les agradezco su tiempo y espero y puedan ayudarme con esta pequeña duda que tengo.

**Comment by LiteBrite82** [comment]
Hi @AlanBrito22 , thanks for participating!

Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting. We’ll be locking any posts in languages other than English for now, including this one.
